torch/_inductor/memory.py

```diff
@@ -93,7 +93,9 @@ def _dep_size_hint(dep: Dep) -> int:
     dep_name_to_size: Dict[str, int] = dict()
     for node in nodes:
         for dep in node.read_writes.reads:
-            if dep.name in graph_inputs and not dep.name.startswith("primals_"):
+            if dep.name in graph_inputs and not (
+                dep.name.startswith("primals_") or dep.name.startswith("arg")
```

Comment:
```suggestion
                dep.name.startswith(("primals_", "arg"))
```
faster, better, more readable.